docs/inventory.md

```diff
@@ -0,0 +1,246 @@
+# NixOS project resource inventory
+
+This is the current list of hardware and services that everyone has access to.
+
+# Accounts
+## GitHub
+
+owner: Eelco, Rob, Domen, Garbas
+
+## Domains
+
+owner: Eelco
+* nixos.org - https://www.uniteddomains.com/
+
+## DNS
+
+owner: Eelco
+Managed manually be Eelco via the United Domains web UI
+
+## AWS account
+
+owner:  Infor
+alias: lb-nixos
+access: Rob and Eelco
+
+## Packet.net
+
+owner: Graham
+
+## Hetzner Cloud
+
+owner: Graham
+(for ofborg)
+
+## IRC logging bot
+
+owner: SamuelDR
```

Comment:
You can use samueldr, or my full name Samuel Dionne-Riel. (I try to keep away from capitalizing the nickname as the "dr" part may cause confusions with the "doctor" title.)